test/lib/mayaUsd/render/mayaToHydra/cpp/testUsdPicking.cpp

```diff
@@ -26,22 +27,19 @@
 #include <ufe/observableSelection.h>
 #include <ufe/globalSelection.h>
 
+#include <vector>
+
 #include <gtest/gtest.h>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
 using namespace MayaHydra;
 
-TEST(TestUsdPicking, pick)
-{
+void pick(const Ufe::Path& selectedPath, const Ufe::Path& markerPath, bool checkNestedInstanceIndices) {
```

Comment:
Extracted out the different pick testing utilities into a common method since there was a lot of overlap (the `testUsdPointInstancePicking.cpp` code was essentially moved here), and now also supports when a UFE path maps to multiple scene index prims.